README.md

```diff
@@ -162,6 +162,16 @@ $.Date :: Type
 
 Type comprising every Date value.
 
+#### `Enum`
+
+```haskell
+$.Enum :: [Any] -> Type
+```
+
+Constructor for [enumerated types][1].
+
+[1]: https://en.wikipedia.org/wiki/Enumerated_type
```

Comment:
`[1]` is already defined in this document. You'll need to juggle the reference ids and slot this link into the list at the bottom of the document.